docs/concepts/pdp-sharding.mdx

```diff
@@ -44,110 +44,222 @@ Permit.io's backend uses that very same sharding algorithm, Thus both the locall
+          connect_timeout: 1s
+          type: STRICT_DNS
+          lb_policy: RING_HASH
+          ring_hash_lb_config:
+            maximum_ring_size: 1024
+          dns_lookup_family: V4_ONLY
+          load_assignment:
+            cluster_name: pdp_cluster
+            endpoints:
+              - lb_endpoints:
+                  - endpoint:
+                      address:
+                        socket_address:
+                          address: pdp-shard-0
+                          port_value: 7000
+                    metadata:
+                      filter_metadata:
+                        envoy.lb:
+                          hash_key: "0"
+                  - endpoint:
+                      address:
+                        socket_address:
+                          address: pdp-shard-1
+                          port_value: 7000
+                    metadata:
+                      filter_metadata:
+                        envoy.lb:
+                          hash_key: "1"
+                  - endpoint:
+                      address:
+                        socket_address:
+                          address: pdp-shard-2
+                          port_value: 7000
+                    metadata:
+                      filter_metadata:
+                        envoy.lb:
+                          hash_key: "2"
+                  - endpoint:
+                      address:
+                        socket_address:
+                          address: pdp-shard-3
+                          port_value: 7000
+                    metadata:
+                      filter_metadata:
+                        envoy.lb:
+                          hash_key: "3"
+                  - endpoint:
+                      address:
+                        socket_address:
+                          address: pdp-shard-4
+                          port_value: 7000
+                    metadata:
+                      filter_metadata:
+                        envoy.lb:
+                          hash_key: "4"
+                  - endpoint:
+                      address:
+                        socket_address:
+                          address: pdp-shard-5
+                          port_value: 7000
+                    metadata:
+                      filter_metadata:
+                        envoy.lb:
+                          hash_key: "5"
+                  - endpoint:
+                      address:
+                        socket_address:
+                          address: pdp-shard-6
+                          port_value: 7000
+                    metadata:
+                      filter_metadata:
+                        envoy.lb:
+                          hash_key: "6"
    ```
 
+2. Perform HTTP request to notify a migration to sharded PDP cluster is being made ( this request must be done with env level api-key ), the response json will contain a `client_secret` that should be used in the next steps as the new cluster PDP_API_KEY:
```

Comment:
what is this doing? will the previous non-sharded PDP be able to still run until the move to shards?

Reply:
yes it doesn't effect the old PDP, it only creates a new one ( with the same name in the same env )